railties/test/generators/scaffold_controller_generator_test.rb

```diff
@@ -221,14 +221,11 @@ def test_model_name_option
     end
 
     assert_file "app/views/admin/users/index.html.erb" do |content|
-      assert_match("'Show', [:admin, user]", content)
-      assert_match("'Edit', edit_admin_user_path(user)", content)
-      assert_match("'Destroy', [:admin, user]", content)
-      assert_match("'New User', new_admin_user_path", content)
+      assert_match("'New user', new_admin_user_path", content)
```

Comment:
```suggestion
      assert_match("\"New user\", new_admin_user_path", content)
```

(double quote fixes the test)

Reply:
test failing:


ScaffoldControllerGeneratorTest#test_model_name_option [test/generators/scaffold_controller_generator_test.rb:223]:
--
  | Expected /'New\ user',\ new_admin_user_path/ to match "<p id=\"notice\"><%= notice %></p>\n\n<h1>User</h1>\n\n<div id=\"users\">\n  <%= render @users %>\n</div>\n\n<%= link_to \"New user\", new_admin_user_path %>\n".